src/pet.js

```diff
@@ -65,5 +66,14 @@ Pet.prototype.checkUp = function() {
         return 'I feel great!'
     }
 }
-   
+
+//Having a child
+Pet.prototype.adoptChild = function(child) {
+    this.children[0] = child;
```

Comment:
`this.children.push` would be better here as would allow you to give the pet more than one child 😃

Reply:
Joe beat me to this :smiley: 
In other words, the method Joe is mentioning will allow you to keep adding more children to the parent, instead of replacing one for the other.